product_docs/docs/pgd/5/consistency/crdt.mdx

```diff
@@ -7,8 +7,8 @@ redirects:
 
 Conflict-free replicated data types (CRDT) support merging values from concurrently modified rows instead of discarding one of the rows as traditional resolution does.
 
-!!! Note Permissions Required
-PGD CRDT require usage access to CRDT types, therefore a user must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
+!!! Note Permissions required
+PGD CRDT requires usage access to CRDT types. Therefore a user must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
```

Comment:
I have the same query here as to why this is in third person.

Reply:
```suggestion
PGD CRDT requires usage access to CRDT types. Therefore any user needint to access CRDT types must have at least the [bdr_application](../security/pgd-predefined-roles/#bdr_application) role assigned to them.
```